src/js/actions/tools.js

```diff
@@ -175,7 +175,7 @@ define(function (require, exports) {
             outsideShiftPolicy = new PointerEventPolicy(adapterUI.policyAction.ALWAYS_PROPAGATE,
                 adapterOS.eventKind.LEFT_MOUSE_DOWN,
                 {
-                    shift: true
+                    shift: false
```

Comment:
So modifiers table is not a true/false table. It's more of an existence thing. 

In this bug's case, you don't want to disable shift, you want shift clicks to all come to us, which you do by changing the propagation action of this policy (to NEVER)